Installation error with Linux Mint

Hello,

I tried installing Mailpile today and I had an error with `sudo make debian-dev` because pip was not installed at some point.

Just had to install python-pip.

Also, after it was installed, I had this output:

```
Gem files will remain installed in /var/lib/gems/1.8/gems/therubyracer-0.12.1 for inspection.
Results logged to /var/lib/gems/1.8/gems/therubyracer-0.12.1/ext/v8/gem_make.out
Fetching: commonjs-0.2.7.gem (100%)
Fetching: less-2.5.0.gem (100%)
Successfully installed commonjs-0.2.7
Successfully installed less-2.5.0
2 gems installed
Installing ri documentation for commonjs-0.2.7...
Installing ri documentation for less-2.5.0...
Installing RDoc documentation for commonjs-0.2.7...
Installing RDoc documentation for less-2.5.0...
make: *** [debian-dev] Error 1
```

And running the command a second time worked. I can make a PR for the python-pip thing and try to look further into it for the Makefile that bugs at the end if needed ? 
Also, I can try the bug on last Ubuntu version.

--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ dev:
 debian-dev:
 	sudo apt-get install python-imaging python-lxml python-jinja2 pep8 \
 	                     ruby-dev yui-compressor python-nose spambayes \
-	                     phantomjs
+	                     phantomjs python-pip
 	if [ "$(shell cat /etc/debian_version)" = "jessie/sid"  ]; then\
 		 sudo apt-get install rubygems-integration;\
 	else \

